bears/xml/XMLBear.py

```diff
@@ -24,10 +24,11 @@ class XMLBear(LocalBear, Lint):
     gives_corrected = True
 
     def process_output(self, output, filename, file):
-        if self.stdout_output:  # only yield Result if stdout is not empty
-            return self._process_corrected(self.stdout_output, filename, file)
-        if self.stderr_output:  # pragma: no cover
+        if self.stderr_output:
+            self.use_stderr = True
```

Comment:
@AbdealiJK use_stderr is needed because if not set to True it will print logs also